contracts/ClaimManager.sol

```diff
@@ -121,9 +126,15 @@ contract ClaimManager is DepositsManager {
     }
 
     require(deposits[claimant] >= minDeposit);
-    require(claimantClaims[claimant] == 0);//claimant can only do one claim at a time
+    // require(claimantClaims[claimant] == 0);//claimant can only do one claim at a time
+    
+//    uint claimId = numClaims;
+//    uint claimId = uint(keccak256(claimant, _plaintext, _hash, numClaims));
+
+    uint claimId = uint(keccak256(claimant, _plaintext, _hash, proposalId));
```

Comment:
I think we could get away with not hashing the plaintext. that way total parameter size is 96 bytes. As opposed to plaintext being dynamically sized

Reply:
Good idea.